src/ReTestItems.jl

```diff
@@ -689,7 +694,9 @@ function manage_worker(
             continue
         end
     end
+    @info "All tests on worker $worker_num completed. Closing $worker."
```

Comment:
Might be helpful to add the number of tests this worker ran on this line?

Reply:
Would be nice to have 

but not utterly trivial to add so moved it to an issue https://github.com/JuliaTesting/ReTestItems.jl/issues/206

as i'd like to merge this PR to get CI back on the road (since this branch includes https://github.com/JuliaTesting/ReTestItems.jl/pull/195 which fixes CI)